pkg/volume/glusterfs/glusterfs.go

```diff
@@ -240,7 +240,12 @@ func (b *glusterfsMounter) setUpAtInternal(dir string) error {
 	if err := os.MkdirAll(p, 0750); err != nil {
 		return fmt.Errorf("glusterfs: mkdir failed: %v", err)
 	}
-	log := path.Join(p, "glusterfs.log")
+
+	// adding log-level ERROR to remove noise
+	// and more specific log path so each pod has
+	// it's own log based on PV + Pod
+	log := path.Join(p, b.pod.Name + "-glusterfs.log")
```

Comment:
the log file is now computed in two places, perhaps this should be a property of the mounter